www/templates/intro/consent.html

```diff
@@ -5,60 +5,78 @@ <h3>Encuesta Movilidad</h3></center>
 </li>
 <li>
-  Su información personal será entregada si es requerido por ley a las entidades correspondientes. Mantendremos sus datos de estudio tan confidencial como sea posible, salvo en el caso de cierta información que debemos informar por razones legales o éticos, como abuso, maltrato o intento de daño a sí mismo o a otros.
+  <span class="bullet"></span>Si usted está de acuerdo participar en la futura encuesta, se le pedirá completar su información de contacto. Se tomarán las mismas medidas descritas arriba para proteger la confidencialidad de los datos proporcionados. Los registros de este estudio tendrán un enlace con el resultado de la futura encuesta, y serán comparados para poder identificar cambios y tendencias en el uso de los medios de transporte. La participación en la futura encuesta, igual que la encuesta actual, es completamente voluntaria.
 </li>
 </ul>
 
 <h3>Uso futuro de datos del estudio</h3>
 Los datos de la investigación se mantendrán para posible uso en el futuro estudio por mí mismo o por otros. Se tomarán las mismas medidas descritas arriba para proteger la confidencialidad de los datos de este estudio.
-Existe una posibilidad de recontactar a usted en el futuro para entender cómo ha cambiado su comportamiento de viajes.
 
 <h3> Derechos </h3>
 <h3> La participación en la investigación es completamente voluntaria. </h3>
 Usted tiene derecho a negarse a participar o retirarse en cualquier momento de este estudio sin penalidad o pérdida de beneficios a los que tenga derecho.
 
 <h3>Preguntas</h3>
-Si tiene alguna pregunta o inquietud acerca de este estudio, favor comunicarse con Megumi Yamanaka en 0981-721493 o <a ng-click="mailTo('megumi_yamanaka@berkeley.edu')">megumi_yamanaka@berkeley.edu</a>.
+Si tiene alguna pregunta o inquietud acerca de este estudio, favor comunicarse con Megumi Yamanaka en 0981-721493 o <a ng-click="mailTo('megumi_yamanaka@berkeley.edu')">megumi_yamanaka@berkeley.edu</a>.<br>
 Si usted tiene alguna pregunta o inquietud acerca de sus derechos y el tratamiento como participante de investigación, puede comunicarse con la oficina del Comité para la Protección de Sujetos Humanos de la Universidad de California, Berkeley, en 1-510-642-7461 o <a ng-click="mailTo('subjects@berkeley.edu')">subjects@berkeley.edu</a>.
 
 <h3> Consentimiento </h3>
-Si usted está de acuerdo en contribuir sus datos a esta investigación, favor hacer click en “Doy mi consentimiento” abajo para continuar. Este aviso está disponible también <a ng-click="openURL('https://drive.google.com/file/d/1jBwcJslxh5GlA8z8tWjTmuPX6wn5yorl/view?usp=sharing')">online</a>
+i usted está de acuerdo en contribuir sus datos a esta investigación, favor hacer click en “Doy mi consentimiento” abajo para continuar. Este aviso está disponible también online <a ng-click="openURL('https://drive.google.com/file/d/1jBwcJslxh5GlA8z8tWjTmuPX6wn5yorl/view?usp=sharing')">(https://goo.gl/AfFGwc)</a>
```

Comment:
Is this intentional? I know very little Spanish, but "Si" -> "i" seems like a typo.